src/include/dns-server/dialog-masterzone.rb

```diff
@@ -207,37 +200,37 @@ def GetMasterZoneEditorTabBasics
       deep_copy(contents)
     end
 
-    def ZoneAclInit
-      allowed = false
-      keys = []
-      Builtins.foreach(Ops.get_list(@current_zone, "options", [])) do |m|
-        if Ops.get_string(m, "key", "") == "allow-transfer" && !allowed
-          key = Builtins.regexpsub(
-            Ops.get_string(m, "value", ""),
-            "^.*\\{[ \t]*(.*)[ \t]*\\}.*$",
-            "\\1"
-          )
+    # @return [Array<String>]
+    def current_zone_allow_transfer
+      items = []
+      @current_zone.fetch("options", []).each do |m|
```

Comment:
Ah, good point. It is in [DnsServerUi.rb](https://github.com/yast/yast-dns-server/blob/05f909f1f94358398feee8f57a5d9fb4cba63f25/src/modules/DnsServerUI.rb#L35) which is an "envelope" module defining the variables and including the parts to work on them, dialog-masterzone.rb is one of them.